src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/persist/SQLParsingUtil.java

```diff
@@ -27,6 +28,10 @@
 public class SQLParsingUtil {
   private static final Logger LOG = LogManager.getLogger(SQLParsingUtil.class);
 
+  public static final SQLParsingUtil API = new SQLParsingUtil();
```

Comment:
do we really need singleton pattern here ? I assume those are just some static utility functions which can be tested directly without singleton.

Reply:
Removed, now tested directly